src/framework/theme/components/calendar-kit/components/calendar-year-picker/calendar-year-cell.component.spec.ts

```diff
@@ -42,6 +42,8 @@ describe('Component: NbCalendarYearCell', () => {
   });
 
   it('should contain cell class', () => {
+    component.date = new Date();
+    fixture.detectChanges();
```

Comment:
Why do we need it? How did it work before?)

Reply:
It worked because `detectChanges` was never called -> no view rendered, just a host element.